modules/twinklespeedy.js

```diff
@@ -248,6 +251,21 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 			}
 		]
 	});
+	tagOptions.append({
+		type: 'checkbox',
+		list: [
+			{
+				label: 'Request creation protection',
```

Comment:
Getting wordy, but I'd like to make it clear this isn't going to post at RfPP or anything.  Might be better ways to phrase it but something like:

```suggestion
				label: 'Tag for creation protection (salting) as well',
```